docs/TROUBLESHOOTING.md

```
@@ -0,0 +1,154 @@
+**Possible causes:**
+
+- The zone is not defined
+- Key variable expansion failed (`$var` is not set)
+- TTL has expired
+- The key has never been written
+
+**How to check:**
+
+Check the error.log for the following messages.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)